lib/maven_project.rb

```diff
@@ -58,7 +58,7 @@ def compiled?
 
   def compile!
     Dir.chdir(path) do
-      SystemCommands.sh!('mvn', '-q', 'package')
+      SystemCommands.sh!('mvn -q package -Dmaven.test.skip=true', {escape: false})
```

Comment:
Ok, so it turns out that I was wrong and you were right - the `escape: false` is not required.